site_source_files/content/blog/2021/selenium-4-observability.md

```diff
@@ -0,0 +1,64 @@
++++
+Description = "Observability in Selenium 4"
+Title = "Observability in Selenium 4"
+Date = 2021-04-26
+Author = "Puja Jagani"
+AuthorLink = "https://twitter.com/pujagani"
+tags = ["selenium", "grid"]
+categories = ["general"]
++++
+
+Selenium server is now power-packed with [Opentelemetry](https://opentelemetry.io/) to help troubleshoot issues and optimize performance. Thus, making the Selenium server observable! 
+
+### Need for Observability 
+
+Selenium server enables distributed testing. Instead of running the test locally, the tests run on a server that makes “**smart**” decisions to run them on different remote servers. Every such remote server is capable of hosting different types of browsers and browser versions. 
```

Comment:
Nit: the tests still run locally. The browsers run remotely. I'm not sure how much it matters, given that I suspect people seldom differentiate the two.